exercise/templates/exercise/_points_badge.html

```diff
@@ -6,27 +6,28 @@
 </span>
 
 {% elif confirm_the_level %}
-{% if submitted %}
-<span class="badge{% if graded and official %} badge-{% if full_score %}success{% elif passed %}warning{% else %}danger{% endif %}{% endif %} {{ classes }}">
-  {% if not official %}{% trans 'Submitted after deadline' %}{% elif passed %}{% trans "Accepted" %}{% elif graded %}{% trans "Rejected" %}{% else %}{% trans "Submitted" %}{% endif %}
-</span>
-{% elif confirmable_points %}
-<span class="badge badge-danger" data-toggle="tooltip" title="{% trans 'You must submit here to confirm your points. Submissions are accepted even after the deadline.' %}">
-  {% trans "Missing" %}
-</span>
-{% elif exercise_page %}
-<span class="badge">
-  {% trans "Not submitted" %}
-</span>
-{% endif %}
-
+    {% if submitted %}
+        <span class="badge{% if graded and official %} badge-{% if full_score %}success{% elif passed %}warning{% else %}danger{% endif %}{% endif %} {{ classes }}">
+        {% if not official %}{% trans 'Submitted after deadline' %}{% elif passed %}{% trans "Accepted" %}{% elif graded %}{% trans "Rejected" %}{% else %}{% trans "Submitted" %}{% endif %}
```

Comment:
This is now handled by below case with `get_points_class` and `get_points_text`, so this part can be refactored in some way so there wouldn't bee too much of duplication.

Reply:
We can add special case for `confirm_the_level` if it's really needed, but I'm pretty sure this is better approach for O1 too. In other words, this rendering should be related to `confirm_the_level`, except it can be used for the extra messages like `Not submitted` and so on (logical as the exercise is required to get points from the chapter).

Reply:
So are you trying to say that I should remove the `confirm_the_level` section from the template? Apparently it has some extra text options which makes it a little harder to just take that section away.

Reply:
Moved most of the logic into the template tags so the points_badge template does not need overly many spans and messy markup.